android/src/main/java/com/phylloconnect/PhylloConnectModule.kt

```diff
@@ -19,7 +19,9 @@ import android.view.View
 import android.content.Intent
 import com.facebook.react.bridge.WritableArray
 import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.ReadableMap
 import com.facebook.react.modules.core.DeviceEventManagerModule
+import kotlin.math.log
```

Comment:
Why is `math.log` required?

Reply:
hmm i don't think so its using in kotlin

Reply:
Then can we remove it?

Reply:
@akshaybhalotia i have removed this one .